src/StatsTools/reweighting.jl

```diff
@@ -69,44 +94,47 @@ w_lin(shift, h, dτ; kwargs...) = w_lin(Vector(shift), h, dτ; kwargs...)
+    )
+    growth_estimator(sim::PMCSimulation; kwargs...)
+    -> r::RatioBlockingResult
+
 Compute the growth estimator with reference energy `E_r` by the reweighting
 technique described in [Umrigar *et al.* (1993)](http://dx.doi.org/10.1063/1.465195),
 see Eq. (20).
-`shift` and `wn` are equal length
-vectors containing the shift and walker number time series, respectively.
-Reweighting is done over `h`
-time steps and `length(shift) - skip` time steps are used for the blocking analysis done
-with [`ratio_of_means()`](@ref). `dτ` is the time step and `weights` a function that
-calulates the weights. See [`w_exp()`](@ref) and [`w_lin()`](@ref).
+`shift` and `wn` are equal length vectors containing the shift and walker number time
+series, respectively.  Reweighting is done over `h` time steps and `length(shift) - skip`
+time steps are used for the blocking analysis done with [`ratio_of_means()`](@ref). `dτ` is
+the time step and `weights` a function that calulates the weights. See [`w_exp()`](@ref) and
+[`w_lin()`](@ref).
 ```math
 E_{gr} = E_r - \\frac{1}{dτ}\\ln
     \\frac{\\sum_n w_{h+1}^{(n+1)} N_\\mathrm{w}^{(n+1)}}
         {\\sum_m w_{h}^{(m)} N_\\mathrm{w}^{(m)}}
 ```
-When `h` is greater than the autocorrelation time scale of the `shift`,
-then `E_gr` (returned as `r.ratio`) is an unbiased but approximate estimator for the ground
-state energy ``E_0`` with an error ``\\mathcal{O}(dτ^2)`` and potentially increased
-confidence intervals compared to the (biased) shift estimator.
-Error propagation is done with [`MonteCarloMeasurements`](https://github.com/baggepinnen/MonteCarloMeasurements.jl). Progagation through the
-logarithm can be modified by setting `change_type` to [`to_measurement`](@ref) in order
-to avoid `NaN` results from negative outliers.
 
-If `success==true` the
-blocking analysis was successful in `k-1` steps, using `blocks` uncorrelated data points.
+When `h` is greater than the autocorrelation time scale of the `shift`, then `E_gr`
+(returned as `r.ratio`) is an unbiased but approximate estimator for the ground state energy
+``E_0`` with an error ``\\mathcal{O}(dτ^2)`` and potentially increased confidence intervals
+compared to the (biased) shift estimator.  Error propagation is done with
+[`MonteCarloMeasurements`](https://github.com/baggepinnen/MonteCarloMeasurements.jl).
+Propagation through the logarithm can be modified by setting `change_type` to
+[`to_measurement`](@ref) in order to avoid `NaN` results from negative outliers.
+
+If `success==true` the blocking analysis was successful in `k-1` steps, using `blocks`
+uncorrelated data points.
 
 The second method calculates the growth estimator directly from a `DataFrame` returned by
-[`lomc!`](@ref Main.lomc!). The keyword arguments `shift_name` and `norm_name` can be used to change the names
-of the relevant columns.
+[`lomc!`](@ref Main.lomc!). The keyword arguments `shift_name` and `norm_name` can be used
+to change the names of the relevant columns.
```

Comment:
Can we reword this to avoid reference to the deprecated `lomc!` and similar to my suggestion for `growth_witness`?